client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mazenet-client",
-  "version": "0.1.0",
+  "version": "0.0.1",
```

Comment:
Not that it matters, but just so you know, the semver docs recommend [starting the development release at 0.1.0](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase).

Reply:
Both the repo and the server's package.json were at 0.0.1, so I brought the client in line since I assumed you set the server to 0.0.1 for a reason. I will update the versions to be 0.1.0

Reply:
oh okay, sounds good